app/models/slide.rb

```diff
@@ -65,6 +65,15 @@ def self.popular_slides(limit = 10)
       limit(limit)
   end
 
+  def self.featured_slides(limit = 10)
+    ids = FeaturedSlide.order(created_at: 'desc').pluck(:slide_id)
+    slides = self.published.
+      includes(:user).
+      where(id: ids).
+      limit(limit)
+    ids.collect {|id| slides.detect {|x| x.id == id.to_i}}
```

Comment:
__[rubocop]__
* Style/SpaceInsideBlockBraces: Space between { and | missing.
* Style/SpaceInsideBlockBraces: Space missing inside }.

<!-- Comment by [SideCI](https://sideci.com) -->